/fault_fracture_localization:
  offboard:
    ros__parameters:
      control_rate: 100 # TrajectorySetpoint broadcase speed in Hz
      target_accel: 3.0 # Target maximum acceleration in any dimension of UAV
      time_lower_bound: 0.0 # Minimum expected amount of time for drone to travel from one waypoint to another
      time_upper_bound: 6000.0 # Maximum expected amount of time for a drone to travel from one waypoint to another

/fault_fracture_localization:
  perception:
    ros__parameters:
      fx: 861.9233198
      fy: 980.985917
      cx: 960.0
      cy: 540.0

/fault_fracture_localization:
  state_machine:
    ros__parameters:
      takeoff_height: 100 # The height the drone should take off to



